datadog_checks_base/tests/base/utils/test_tls.py

```diff
@@ -45,11 +45,18 @@ def test_config(self):
 
     @pytest.mark.parametrize('param', ('tls_ca_cert', 'tls_cert', 'tls_private_key', 'tls_private_key_password'))
     def test_config_overwrite(self, param):
-        config = {'tls_verify': False, param: 'foo'}
+        config = {param: param}
```

Comment:
You should keep `tls_verify: False` here because the default is True.
And remove all but `tls_ca_cert` from the parametrize decorator